app/utils/format-config.js

```diff
@@ -0,0 +1,15 @@
+import formatSha from 'travis/utils/format-sha';
+
+const slugRe = (slug) => (new RegExp(`^${slug}/`));
+
+const fileName = (source) => (source.match(/([^\/]*)(@\w{16})?$/)[1]);
+
+export const fileNameWithoutSha = (source) => (fileName(source).replace(/(@\w{16})$/, ''));
```

Comment:
It seems like `.replace` is doing nothing here, because `fileName` function already cleans up the hash, isn't it?

Also please use function declarations instead of function expressions for exported functions:
```js
export function fileNameWithoutSha(source = '') {
  return fileName(source);
}
```
Function declarations have some advantages over lambdas, e.g. function name, which my simplify debugging.

Reply:
Maybe even just export `fileName` function calling it something like extractFileName? What do you think?

Reply:
About 1st one nope because fileName returns name with @ and sha